controller/Review.php

```diff
@@ -298,4 +312,51 @@ private function isSubmissionReviewRequestMessageProvided(): bool
 
         return $messageType === LtiMessageInterface::LTI_MESSAGE_TYPE_SUBMISSION_REVIEW_REQUEST;
     }
+
+    private function getReviewPanelConfig(): array
+    {
+        $extensionsManager = $this->getServiceLocator()->get(common_ext_ExtensionsManager::SERVICE_ID);
+        $extension = $extensionsManager->getExtensionById('ltiTestReview');
+        return $extension->getConfig('ReviewPanel');
+    }
+
+    private function getDisplaySectionTitlesOption(LtiLaunchData $launchData): bool
```

Comment:
The name of this method seems a bit misleading. I'd expect to have the options in return but it seems it's enable/disable the section titles.

What do you think about this?

```suggestion
    private function hasSectionTitlesEnabled(LtiLaunchData $launchData): bool
```

Reply:
It's actually named that way to reflect the LTI param. For example:
`show_score` -> `getShowScoreOption`
`show_correct` -> `getShowCorrectOption`
`section_titles` -> `getDisplaySectionTitlesOption` // LTI param was shortened for convenience but it's still `displaySectionTitles` internally
`review_layout` -> `getReviewLayoutOption`

Does it make sense?